### DISCUSSION TITLE
Configuration file path

### DISCUSSION THREAD

**Discussion by y-luis-rojo** [tchiotludo/akhq#1565] [Q&A] [answered]
From https://akhq.io/docs/installation.html, for a Docker installation, I assume configuration file (application.yml) path must rest in the same folder as the application (/app) is that correct? If so, is there a way to change the configuration file path (using environment variables maybe)?

**Comment by tchiotludo** [comment, marked as answer]
you can use `MICRONAUT_CONFIG_FILES` to target any path for a configuration files

**Comment by y-luis-rojo** [comment]
Exactly what I needed. Thanks.

**Reply by y-luis-rojo** [reply]
Just a suggestion: if it is not documented (I was not able to find it) it should be.
Also, worth to mention that the path must include the file (i.e.: `MICRONAUT_CONFIG_FILES=/somepath/application.yml`).

**Reply by tchiotludo** [reply]
PR are welcome :tada:

**Reply by y-luis-rojo** [reply]
Done: https://github.com/tchiotludo/akhq/pull/1566
